brainscore_language/artificial_subject.py

```diff
@@ -41,11 +41,11 @@ class Task:
         .. code-block:: python
         
            {'behavior': <xarray.BehavioralAssembly (presentation: 3)>
-                        array(['fox', 'over', 'dog']), # the actual next words
+                        array(['fox']), # the actual next words
                         Coordinates:
                           * presentation  (presentation) MultiIndex
-                          - context       (presentation) object 'the quick brown' 'fox jumps', 'over the lazy'
-                          - stimulus_id   (presentation) int64 0 1 2}
+                          - context       (presentation) object 'the quick brown fox jumps over the lazy'
+                          - stimulus_id   (presentation) int64 0}
         """
```

Comment:
As per conversation with Martin, updated documentation since mult-text input is concatenated under the hood, and produces just one output

Reply:
I think we talked past each other -- the above use case should still be supported.
What I was trying to get at was:
1. if a list of text is supplied to the model, the model produces one output per text item, *maintaining* its internal state (if there is one). When we talked about it, I thought you just meant you can iterate inside the model
2. upon subsequent calls to the `digest_text` method, reset any internal state in between